### PR TITLE
TOC and file renaming

### PR DESCRIPTION
**Pull request by root-hal9000**
- renamed all files to be lowercase and have no spaces
- Updated all references to those names in file content
- Recreated table of contents into readme.MD Issue #2
- Separated "importing docs" text into a separate file
- Added a few %%{WARNING}%% tags to links that go to zadams repos. Issue #87

### CODE REVIEW

**Comment by Alumniminium** [comment]
seems to have overwritten my changes to the web-clipper.md, could you pull that into your merge request please :)

File: Wiki/web-clipper.md
Comment (Alumniminium) [inline Wiki/web-clipper.md:1]: Please copy the contents from https://github.com/TriliumNext/Docs/blob/main/Wiki/Web-clipper.md

**Review by Alumniminium — CHANGES_REQUESTED** [review]
Please delete the old files with MixedCase.

**Comment by root-hal9000** [comment]
> Please delete the old files with MixedCase.

Oops, sorry about that, they had been deleted (I originally renamed them with a bash script) but I had issues commiting and re-cloned to copy my changes over and didn't notice that. Will do

**Comment by root-hal9000** [comment]
OK, deleted the old files - anything else here or should I merge. If someone merges, create a new initial-review branch from main after merging this

**Comment by maphew** [comment]
was all of this work scripted? I presume yes or mostly. Perhaps put them in `/utils` or something, or at least keep the transition scripts handy in the event there's more changes upstream. I don't think there's a high probability of that, but have been bitten many times in the past by the thought "we'll only need to do this once" ... ;-)

**Comment by root-hal9000** [comment]
> was all of this work scripted? I presume yes or mostly.
Only the renaming of the files to lowercase/no space (and replacing of the links referring to them). The TOC was manual as there's no inherent structure in the files for that and I made some slight changes to the order

**Review by Alumniminium — APPROVED** [review]

**Comment by Alumniminium** [comment]
oh god i should have merged that from the terminal and not from the web... what just happened

**Comment by root-hal9000** [comment]
> oh god i should have merged that from the terminal and not from the web... what just happened

Just got out of the meeting, not sure what you're referring too although I see a couple of new branches?

Also, I should have made the PR to the main branch, no? We should merge this into main, and then pull the initial-review branch from there

**Comment by Alumniminium** [comment]
> [...] Also, I should have made the PR to the main branch, no? We should merge this into main, and then pull the initial-review branch from there

Feel free to do that, otherwise I'll do that tonight after work.

**Comment by Alumniminium** [comment]
Done. I've merged that branch into main and created an 'initial-review' branch from the new main. @root-hal9000

**Pull request merged**